WebApp/src/components/BlockDetail/BlockDetail.js

```diff
@@ -184,8 +190,8 @@ class BlockDetail extends PureComponent {
               <ClearIcon nativecolor={color} />
             </IconButton>
           </div>
-          <div className="name" style={{ color }}>
-            {course.courseName}
+          <div className="name" style={{ color }} >
+            {this.textTrans(course.courseName)}
```

Comment:
If textTrans is used only here and nowhere else than I think u can use { textTransform(course.courseName) }. Pretty sure it works but if u need function then nothing.